bauble_macros/tests/derive.rs

```diff
@@ -145,39 +145,6 @@ fn test_std_types() {
     );
 }
 
-#[test]
-fn test_complex_flatten() {
```

Comment:
This test did not work with my changes, however, I realized it was relying on the copy feature. Which is a broken feature and is underway to be removed. This does not affect the main codebase and it seemed safe to remove this test.